app.js

```diff
@@ -10,6 +14,16 @@ app.get("/api/topics", returnTopics);
 
 app.get("/api/articles/:article_id", returnArticle);
 
+app.patch("/api/articles/:article_id", addVotesToArticle);
+
+app.get("*", function (req, res) {
+  res.status(404).send({ msg: "Page not Found" });
+});
+
+app.patch("*", function (req, res) {
+  res.status(404).send({ msg: "Page not Found" });
+});
```

Comment:
This isn't how your errors should be handled... They're handled in the function below.